api/src/main/java/io/opencensus/common/Functions.java

```diff
@@ -53,6 +53,14 @@ public Void apply(Object ignored) {
         }
       };
 
+  private static final Function<Object, /*@Nullable*/ String> RETURN_TO_STRING =
+      new Function<Object, /*@Nullable*/ String>() {
+        @Override
+        public String apply(Object input) {
+          return input.toString();
```

Comment:
should this be:

return input == null ? null : input.toString()?

Reply:
Updated.